lib/library_web/controllers/page_controller.ex

```diff
@@ -1,7 +1,30 @@
 defmodule LibraryWeb.PageController do
   use LibraryWeb, :controller
+  import Library.Books
+  alias Library.GoogleBooks
 
   def index(conn, _params) do
-    render conn, "index.html"
+    books = list_books()
+
+    render conn, "index.html", books: books, web: false
```

Comment:
Can we keep with the same format for `render` across the board? It's clearest with parentheses.